projects/gnomad/data/export_exac_vcf_to_ht.py

```diff
@@ -0,0 +1,528 @@
+# Usage:
```

Comment:
How should we document loading script usage going forward? I greatly prefer it to be executable through a Makefile directive, but I realize this comes with the cost of keeping track of file paths/variables.